types/nodemailer-ses-transport/index.d.ts

```diff
@@ -1,22 +1,17 @@
-// Type definitions for nodemailer-ses-transport 1.4
+// Type definitions for nodemailer-ses-transport 3.1
```

Comment:
Hi. I just noticed that the new version doesn't match mayor/minor of any [published version of the package](https://www.npmjs.com/package/nodemailer-ses-transport).

Is this a mistake?
Shouldn't we downgrade it to 1.5?

Reply:
I created [PR 51291](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/51291) to fix this.